ara/cli/host.py

```diff
@@ -65,8 +67,30 @@ def get_parser(self, prog_name):
         return parser
 
     def take_action(self, parsed_args):
-        hosts = (models.Host.query
-                 .filter((models.Host.id == parsed_args.host) |
-                         (models.Host.name == parsed_args.host)).one())
+        host = (models.Host.query
+                .filter((models.Host.id == parsed_args.host) |
+                        (models.Host.name == parsed_args.host)).one())
+
+        return utils.fields_from_object(FIELDS, host)
+
+
+class HostFacts(ShowOne):
+    """Show facts for a host"""
+    log = logging.getLogger(__name__)
+
+    def get_parser(self, prog_name):
+        parser = super(HostFacts, self).get_parser(prog_name)
+        parser.add_argument(
+            'host',
+            metavar='<host>',
+            help='Host name or id to show facts for',
+        )
```

Comment:
I think we should accept fact names as optional arguments:

    parser.add_argument(
        'fact',
        metavar='<fact>',
        help='Optional name of a specific fact to display',
        nargs='*',
    )


...and then use that to limit the facts that are displayed.  This is
handy if you want to get a particular value, e.g:

    ara host facts host1 ansible_os_family -f shell

Or:

    ara host facts host1 ansible_interfaces -f json